tests/languages/elixir/keyword_feature.test

```diff
@@ -6,6 +6,8 @@ defimpl defmodule
 defp
 defprotocol
 defdelegate
+defmacro quote unquote
+defn defnp

```

Comment:
The expected JSON (the one below) also has to be updated. Just run `npm run test:languages -- --update`.

Reply:
Thank you - should be fixed in 275542d